spread.yaml

```diff
@@ -607,7 +607,7 @@ exclude:
     - "*.a"
     - "*.pyc"
     - ./vendor
-    - "*.snap"
+    - "snapd_*.snap"
```

Comment:
revert this please
"*.snap" is very important to avoid we include any pre-built / downloaded snap

Reply:
fixed